packages/flare/bin/data_store.py

```diff
@@ -0,0 +1,110 @@
+import configparser
+import os
+
+from constants import APP_NAME
+from constants import DataStoreKeys
+from datetime import datetime
+from typing import Optional
+
+
+# Define the config file path
+splunk_home = os.environ.get("SPLUNK_HOME", "/opt/splunk")
+config_path = os.path.join(
+    splunk_home, "etc", "apps", f"{APP_NAME}", "local", "data_store.conf"
+)
+
+
+class ConfigDataStore:
+    _store: configparser.ConfigParser
```

Comment:
We don't really do that in python.

If you want to define the type for the field, you can do it in the constructor.

```
def __init__(self) -> None:
    self.field: int = 1
```